### ISSUE TITLE
[Feature/Cloud Device Farm]: Screen Functionality on Google Cloud Run Research

### ISSUE THREAD

**Issue by oneofthezombies** [dogu-team/dogu#454]
### Problem

We need to make sure that Google Cloud Run can provide screen functionality.
- Device screen streaming and recording
- Window (app) streaming and recording

### Solution brainstorm

Configure and verify the environment for testing in Google Cloud Run.

**Comment by oneofthezombies** [comment]
Depends on https://github.com/dogu-team/dogu/issues/452
We must verify the platform and architecture for running the Dogu Agent before configuring the test environment.

**Comment by oneofthezombies** [comment]
- I emulated a graphics device in a Google Cloud Run environment using xvfb.
- Verified running without additional options for Chrome and Firefox browsers.
- Recorded at 1920x1080 resolution with ffmpeg for 15 seconds.
- I got out of memory while recording ffmpeg, so I changed the memory from 512MB to 1GB. I used the CPU as 1 core.

https://github.com/dogu-team/dogu/assets/33236451/813bf43e-f293-4b87-9399-a6187f9f6a2b


References
https://www.x.org/releases/X11R7.6/doc/man/man1/Xvfb.1.xhtml

**Comment by oneofthezombies** [comment]
I have confirmed that the browser is running and screen recording is possible.
It has not been confirmed that the Android emulator and the screen control module within the DoguAgent work.
As per the initial specification, we discussed verbally to exclude Android emulators,
I think I need to run DoguAgent on Google Cloud Run and see if there are any issues.
We will proceed with this issue at https://github.com/dogu-team/dogu/issues/455 .
